backend/router/dboperations.py

```diff
@@ -0,0 +1,184 @@
+import datetime
```

Comment:
move over db operations to a different db utility file. create separate routers for members, chapters